### PR TITLE
Clean up components

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

**Comment by bladicito** [comment]
@lehmi11, I am not sure about the use client directives. maybe we need to remove them all of the design. 
All is now clean. We can merge this pr.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ost-cas-fee-adv-23-24/elbmum-design",
-  "version": "0.1.0",
+  "version": "1.35.0-beta3",
```
Comment (lehmi11) [inline package.json]: is this beta version name still needed?

File: src/components/input-group/utils/inputFielfGroup.enum.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export enum EInputFieldGroupType {
```
Comment (lehmi11) [inline src/components/input-group/utils/inputFielfGroup.enum.ts:1]: is there a benefit of defining all possible values in an enum or is it just personal preference?

File: src/components/tabs/Tabs.tsx [hunk 3]
```diff
@@ -1,16 +1,86 @@
...
 
-export interface ITabsProps {
-  active?: boolean;
-  children: ReactNode;
-}
+import { TabsItem } from './TabsItem';
+import { ITabsProps } from './utils/tabs.interface';
+
+export const Tabs = ({ tabItems, updateSelection }: ITabsProps) => {
+  const [tabWidths, setTabWidths] = useState<
```
Comment (lehmi11) [inline src/components/tabs/Tabs.tsx:7]: what is this width-state for? is it used for the animation?

**Review by lehmi11 — APPROVED** [review]
Wow that is a huge refactoring 😮 Nice work 💪🏻

File: package.json (hunk 1 above)
Comment (bladicito) [reply package.json]: We ca revert this value 👍🏽

File: src/components/input-group/utils/inputFielfGroup.enum.ts (hunk 2 above)
Comment (bladicito) [reply src/components/input-group/utils/inputFielfGroup.enum.ts:1]: Well, Personal preference. Also wanted to keep the consistency within all components. 
I find it more elegant even if it produces a bit more code.

File: src/components/tabs/Tabs.tsx (hunk 3 above)
Comment (bladicito) [reply src/components/tabs/Tabs.tsx:7]: Yes, we need to know how big each tab is in order to move the effect in the correct position. They are calculated one on rendering and they should not change anymore.

**Pull request merged**